### ISSUE TITLE
please block this model which is abnormal.

### ISSUE THREAD

**Issue by taozhiyuai** [ollama/ollama#4980]
### What is the issue?

https://ollama.com/rouge/qwen2-7b-instruct-deccp:latest

please block this model, which is not uncensored model, but a model with political purpose. please block this model. we love ollama which is for model lover only.

### OS

macOS

### GPU

Apple

### CPU

Apple

### Ollama version

0.1.32

**Comment by AncientMystic** [comment]
Personally i think everyone should have a voice and we cannot pick and choose who can and cannot speak many models are highly bias and very political if you choose to interact with them on those subjects so i do not see how voicing the matter is much different than all the companies quietly enforcing their own views as a standard of "morality"

**Comment by AncientMystic** [comment]
If everything someone somewhere does not agree with is removed nothing will be left, someone somewhere always disagrees with everything. 

But i do not speak for the ollama team in any way.

**Comment by taozhiyuai** [comment]
every morning you only wash dirty from your face, which will not result in a you with your face. can you identify which part should be cleaned?

I love ollama which is helpful for me. that model is not a model as an AI.  we have brain to identify real model and which one is dirty. Good and helpful model is welcomed and remained.

keep thought logical and rational

**Comment by AncientMystic** [comment]
If you do not see it as productive or useful for you, if you do not like it, do not use it. 

why must others be restricted from using what you decide is not useful for you or liked by you? 

There are many models I personally do not like, so i do not use them, others like them and thats great, we are all free to use the models we like, that is why we have so many wonderful options and everyone is constantly making more, it is also kind of the purpose of ollama, to be able to choose your model and how it is used, privately according to your own wishes.

**Comment by taozhiyuai** [comment]
it is not a AI model, which is just trained by the data author want to say, whose purpose is not for AI. it is obviously.

**Comment by AncientMystic** [comment]
It seems to just be trained on data that could be otherwise disagreeable, many models have preset responses and restrictions they will not deviate from based very heavily upon political beliefs usually under the pretences of that view being of a superior morality or ethical concerns and i agree setting specific hardcoded responses really does violate the spirit of AI, i wish that no one did that, as we can never hope to get completely accurate information from highly restricted models they instead become very complicated and costly ad campaigns for a person or more often a companies viewpoints instead of an ai model. 

but sadly if you test models enough you will find almost every single model has them, it's actually very hard to find models that do not. 

Silencing one viewpoint in the end wouldn't help fix this issue but only perpetuate a one sided narrative. 

If you have legal concerns surrounding that model for those living in the country of china, that may be a better position to address.

**Comment by daniel-dona** [comment]
> it is not a AI model, which is just trained by the data author wants to say, whose purpose is not for AI. it is obviously.
> 
> yes. I agree I can just ignore it. But I do not want to lose ollama which is a website focus on AI models, should not be abused by someone who does not share the same vision with us who focus on AI!

So you want to combat censorship with more censorship?

**Comment by AncientMystic** [comment]
> > it is not a AI model, which is just trained by the data author wants to say, whose purpose is not for AI. it is obviously.
> 
> > 
> 
> > yes. I agree I can just ignore it. But I do not want to lose ollama which is a website focus on AI models, should not be abused by someone who does not share the same vision with us who focus on AI!
> 
> 
> 
> So you want to combat censorship with more censorship? 

That was kind of my point too i mean, the person who made it seems to be trying to add data in an attempt to combat censorship in a way and then we could call that censorship of the accepted narrative setup by the creators of the model and now we see calls for censorship to censor the content that is censoring the views of censorship, wow, what a spiral... 

Everyone should just be free to learn what they want and use what they want and whatever is most accurate and acceptable will get used. Censorship is never the answer.

**Comment by daniel-dona** [comment]
> > > it is not a AI model, which is just trained by the data author wants to say, whose purpose is not for AI. it is obviously.
> > 
> > 
> > > 
> > 
> > 
> > > yes. I agree I can just ignore it. But I do not want to lose ollama which is a website focus on AI models, should not be abused by someone who does not share the same vision with us who focus on AI!
> > 
> > 
> > So you want to combat censorship with more censorship?
> 
> That was kind of my point too i mean, the person who made it seems to be trying to add data in an attempt to combat censorship in a way and then we could call that censorship of the accepted narrative setup by the creators of the model and now we see calls for censorship to censor the content that is censoring the views of censorship, wow, what a spiral...
> 
> Everyone should just be free to learn what they want and use what they want and whatever is most accurate and acceptable will get used. Censorship is never the answer.

It would be better to evaluate independently all the published models and show the information in the model zoo pages... Not only about censorship but also for gender bias and other equality concerns.

**Comment by AncientMystic** [comment]
> > > > it is not a AI model, which is just trained by the data author wants to say, whose purpose is not for AI. it is obviously.
> 
> > > 
> 
> > > 
> 
> > > > 
> 
> > > 
> 
> > > 
> 
> > > > yes. I agree I can just ignore it. But I do not want to lose ollama which is a website focus on AI models, should not be abused by someone who does not share the same vision with us who focus on AI!
> 
> > > 
> 
> > > 
> 
> > > So you want to combat censorship with more censorship?
> 
> > 
> 
> > That was kind of my point too i mean, the person who made it seems to be trying to add data in an attempt to combat censorship in a way and then we could call that censorship of the accepted narrative setup by the creators of the model and now we see calls for censorship to censor the content that is censoring the views of censorship, wow, what a spiral...
> 
> > 
> 
> > Everyone should just be free to learn what they want and use what they want and whatever is most accurate and acceptable will get used. Censorship is never the answer.
> 
> 
> 
> It would be better to evaluate independently all the published models and show the information in the model zoo pages... Not only about censorship but also for gender bias and other equality concerns. 

I agree it would be better to just allow everyone to list concerns of any kind then each user can evaluate on their own whether the concern is a positive or negative for them or their use case personally. Everyone has different beliefs and opinions and no one wants to be on the other side of censorship. 

I mean what if you want to use a model to write a story for example and you want a character or something and the model will NOT violate its programming to play that character or take on its persona to generate that viewpoint because "that's bad" but thats the whole point of the character..... 

There are many reasons to want to generate content besides "if you don't agree you're bad and think bad things" as people tend to see it. It is pretty wild how strictly models are made to only hold one view and never ever contradict it. 

The ai is supposed to do what we tell it to.
